Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "rustls-pki-types"
-version = "1.0.1"
+version = "1.1.0"
```

Comment:
For crates that have a non-zero major version, ideally we should bump the minor version for new features. This is not as strict of a cultural norm in the crates ecosystem as the semver-compatibility rules, but it can help to distinguish when you can downgrade (I'm a little fuzzy on the details right now).